docs/reference/migration/migrate_7_6.asciidoc

```diff
@@ -14,12 +14,42 @@ See also <<release-highlights>> and <<es-release-notes>>.
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
+[discrete]
+[[breaking_76_security_changes]]
+=== Security changes
+
+[discrete]
+==== {es} API key privileges
+
+If you use an API key to create another API key (sometimes called a
+_derived key_), its behavior is impacted by the fix for
+https://www.elastic.co/community/security[CVE-2020-7009].
+
+When you make a request to create API keys, you can specify an expiration and
+privileges for the API key. Previously, when you created a derived key, it had
+no privileges. This behavior disregarded any privileges that you specified in
+the {ref}/security-api-create-api-key.html[create API key API].
+
+As of 7.6.2, this behavior changes. To create derived keys with no privileges,
+you must explicitly specify an empty object in the `role_descriptors` array.
```

Comment:
The `role_descriptors` is technically an object not an array.

Reply:
Thanks, I'll fix that in the API ref in a separate PR too.  For here, I've changed the sentence to "...you must explicitly specify an empty role descriptor".